utils/fo-installdeps

```diff
@@ -85,6 +107,31 @@ CODENAME=$(lsb_release --codename --short)
 
 echo "install core dependencies"
 
+# Darwin
+if [[ $BUILDTIME && "$(uname)" == "Darwin" ]]; then
+   echo "*** Installing buildtime dependencies ***";
+   brew install curl libxml2 icu4c cmake ninja boost postgresql composer
+   brew install mxml cunit libmagic glib libpq cppunit
+
+   case "$CODENAME" in
+       stretch|buster|bullseye|bookworm|sid|focal|jammy)
```

Comment:
Should be replaced with correct codenames for Darwin?

Reply:
However, should not effect for homebrew?????

Reply:
No, according to the current code, it would not affect `Homebrew`. The `codenames` are used to handle specific Linux distributions like `Debian` and `Ubuntu`. Replacing `codenames` for `Darwin` would involve identifying version-specific actions for different macOS versions.

Therefore, I decided to proceed without adding complexity to the script, as the package manager can automatically detect dependencies.